web/src/app/alerts/CreateAlertDialog/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+import {
+  Dialog,
+  Stepper,
+  Step,
+  StepLabel,
+  DialogContent,
+} from '@material-ui/core'
+import { isWidthDown } from '@material-ui/core/withWidth'
+import DialogNavigation from './DialogNavigation'
+import StepContent from './StepContent'
+import { FormContainer } from '../../forms'
+import DialogTitleWrapper from '../../dialogs/components/DialogTitleWrapper'
+import useWidth from '../../util/useWidth'
+import useCreateAlerts from './useCreateAlerts'
+
+export default props => {
+  const width = useWidth()
+
+  const [activeStep, setActiveStep] = useState(0)
+  const [formFields, setFormFields] = useState({
+    // data for mutation
+    summary: '',
+    details: '',
+    selectedServices: [],
+
+    // form helper
+    searchQuery: '',
+  })
+
+  const [
+    createAlerts,
+    { data: alertsCreated, error: alertsFailed, loading: isCreatingAlerts },
+  ] = useCreateAlerts(formFields)
+
+  const onStepContentChange = e => {
+    setFormFields(prevState => ({ ...prevState, ...e }))
+  }
+
+  const steps = ['Alert Info', 'Service Selection', 'Review', null]
+
+  const onLastStep = () => activeStep === steps.length - 1
+
+  const onClose = () => {
+    props.handleRequestClose()
+    // NOTE dialog takes time to fade out
+    setTimeout(() => {
+      setActiveStep(0)
+      setFormFields({
+        summary: '',
+        details: '',
+        selectedServices: [],
+        searchQuery: '',
+      })
+    }, 1000)
+  }
+
+  return (
+    <Dialog
+      open={props.open}
+      onClose={onLastStep() ? null : onClose} // NOTE only close on last step if user hits Done
+      fullScreen={isWidthDown('md', width)}
+      fullWidth
+      width={'md'}
+    >
+      <DialogTitleWrapper
+        fullScreen={isWidthDown('md', width)}
+        title={'Create New Alert'}
+      />
+      <DialogContent style={{ height: '500px' }}>
```

Comment:
We should probably set the height on the dialog itself, and use `vh` so it is dynamic based on the user's height, as well as make sure that height isn't set if we're in fullscreen (`isWidthDown('md', width)`)